app/controllers/admin/overviews_controller.rb

```diff
@@ -90,6 +134,81 @@ def services_to_export
       end
     end
 
+    def service_summary(env)
+      published_service_ids = published(env).reject { |p| moj_forms_team_service_ids.include?(p.service_id) }.map(&:service_id)
+      result = []
+
+      published_service_ids.each do |id|
+        metadata = MetadataApiClient::Service.latest_version(id)
+        result << {
+          service_id: id,
+          service_name: metadata['service_name'],
+          locale: metadata['locale'],
+          page_counts: page_type_counts(metadata['pages']),
+          component_counts: component_type_counts(metadata['pages']),
+          confirmation_email_enabled: ServiceConfiguration.find_by(service_id: id, deployment_environment: env, name: 'CONFIRMATION_EMAIL_COMPONENT_ID').present?,
```

Comment:
A suggestion for performance.

Instead of finding each individual configuration, we can retrieve all of them at once with:
`features = ServiceConfiguration.where(service_id: id, deployment_environment: env).pluck(:name)`

Then we just check if the array of features contains the one we want:
`features.include?('CONFIRMATION_EMAIL_COMPONENT_ID')`